go/processor/floria/run_context.go

```diff
@@ -20,6 +20,8 @@ import (
 	"github.com/ethereum/go-ethereum/crypto"
 )
 
+var emptyCodeHash = tosca.Hash(crypto.Keccak256(nil))
```

Comment:
global variable to be used just once ?

Reply:
Yes, because I don't want to recompute the hash every time the function is called.